helix-term/src/commands.rs

```diff
@@ -1030,20 +1030,29 @@ fn goto_file_impl(cx: &mut Context, action: Action) {
         .map(|r| text.slice(r.from()..r.to()).to_string())
         .collect();
     let primary = selections.primary();
-    if selections.len() == 1 && primary.to() - primary.from() == 1 {
```

Comment:
The condition on `selections.len()` is important here - it's checking that there is only one range (`primary`) in the selection

Reply:
Ahhn, I understand now! Thanks